docs/launchpad/guides/monitoring-stack-with-HA.md

```diff
@@ -0,0 +1,699 @@
+---
+---
+# Deploying a Monitoring stack with HA
+
+## Prerequisites
+- A fully functional working Kubernetes cluster
+- Two object storage buckets: one for Logs data, with Loki, and one for Metrics data, with Thanos
+
+## Configuring Loki for HA
+
+Launchpad uses the [loki-distributed](https://github.com/grafana/helm-charts/tree/main/charts/loki-distributed) release for setting up Loki, which can be configured according to its values interface (as seen [here](https://github.com/grafana/helm-charts/blob/main/charts/loki-distributed/values.yaml))
+
+*Note*: The example setups we'll show will be based on an architecture that makes use of the following components: querier, distributor, ingester, queryFrontend, gateway, compactor, ruler, indexGateway. Different architectures are possible so adjust to your needs as necessary.
+
+For an HA one thing that we will need to do is setup the several components with multiple replicas each, and loki-distributed values can be set for that purpose like in the following example snippet:
+```yaml
+querier:
+  replicas: 2
+  maxUnavailable: 1
+distributor:
+  replicas: 3
+  maxUnavailable: 2
+ingester:
+  replicas: 3
+  maxUnavailable: 2
+queryFrontend:
+  replicas: 2
+  maxUnavailable: 1
+gateway:
+  replicas: 2
+  maxUnavailable: 1
+compactor:
+  kind: Deployment
+  replicas: 1
+  enabled: true
+ruler:
+  enabled: true
+  replicas: 2
+  maxUnavailable: 1
+indexGateway:
+  enabled: true
+  replicas: 2
+  maxUnavailable: 1
+loki:
+  structuredConfig:
+    ruler:
+      ring:
+        kvstore:
+          store: memberlist
+    ingester:
+      lifecycler:
+        ring:
+          replication_factor: 2
+```
+
+*Warning*: If you use a compactor there should only be at most one instance of it.
```

Comment:
```suggestion
*Warning*: If you use a compactor there should only be at most one instance of it to avoid conflicts or redundant work when deduplicating and compacting logs
```